src/dune

```diff
@@ -1,10 +1,28 @@
+;; `melange.emit` is a Dune stanza that will produce build rules to generate
+;; JavaScript files from sources using the Melange compiler
+;; https://dune.readthedocs.io/en/stable/melange.html#melange-emit
 (melange.emit
+
+ ;; The `target` field is used by Dune to put all JavaScript artifacts in a specific folder inside `_build/default`
+ ;; This field `target` allows to have multiple `melange.emit` stanzas in the `dune` file
+ ;; if, for example, we want to produce ES6 and CommonJS output from the same sources.
  (target output)
+
+ ;; The `alias` field can be used later on to build just the artifacts from this `melange.emit`
+ ;; by calling `dune build @react`
```

Comment:
I would add `By default the @melange alias is created`

Reply:
Meh, is on the docs.

Reply:
https://github.com/melange-re/melange-opam-template/pull/26/commits/cf844eeadb76c103f7a49b4b8d35a8455c8dd86e